modules/bag/src/zenoh_player.cpp

```diff
@@ -173,14 +174,14 @@ void ZenohPlayer::Impl::run() {
     }
 
     auto success = publisher->publish({ message.message.data, message.message.dataSize });
-    LOG_IF(WARNING, !success) << fmt::format("Failed to publish message {} for topic {}",
-                                             message.message.sequence, topic);
+    heph::logIf(heph::WARN, !success, "failed to publish message", "message", message.message.sequence,
+                "topic", topic);
 
     ++msgs_played_count;
   }
 
-  LOG(INFO) << fmt::format("Played {} messages, missed {} deadlines", msgs_played_count,
-                           deadline_missed_count);
+  heph::log(heph::INFO, "playing finished", "num_played_messages", msgs_played_count, "num_missed_deadlines",
```

Comment:
we always try to use `xxx_count` instead of `num`
```suggestion
  heph::log(heph::INFO, "playing finished", "played_message_count", msgs_played_count, "num_missed_deadlines",
```